apps/hash-ai-worker-ts/src/activities/infer-entities/create-entities.ts

```diff
@@ -81,7 +92,13 @@ export const createEntities = async ({
         (proposedEntities ?? []).map(async (proposedEntity) => {
           const { properties = {} } = proposedEntity;
 
-          const nameProperties = ["name", "display-name", "preferred-name"];
+          const nameProperties = [
+            "name",
+            "display-name",
+            "legal-name",
+            "preferred-name",
+            "profile-url",
+          ];
```

Comment:
Think I asked previously, wondering why we don't include `title` in this list
```suggestion
          const nameProperties = [
            "name",
            "display-name",
            "legal-name",
            "preferred-name",
            "profile-url",
            "title",
          ];
```

Reply:
My worry with `title` is that it's also used for things that are less likely to be unique, for example a job title, or Mr/Mrs